package.json

```diff
@@ -47,13 +48,14 @@
   "devDependencies": {
     "@nuxt/eslint": "^1.4.1",
     "@nuxt/test-utils": "^3.17.5",
-    "@nuxtjs/i18n": "^9.5.5",
+    "@nuxtjs/i18n": "^9.5.6",
```

Comment:
Was this bump necessary for this PR?

Reply:
It's fine to keep this, but assuming it wasn't necessary to bump `i18n`, I'd keep this for another PR in the future just to stay well-scoped and avoid any minor debugging / creep issues that are actually not related to the task at hand.